web/docs/project/env-vars.md

```diff
@@ -104,43 +206,70 @@ Defining environment variables in this way can be cumbersome even for a single p
 
 ## Defining Env Vars in Production
 
-While in development, we had the option of using `.env.client` and `.env.server` files which made it easy to define and manage env vars.
```

Comment:
Aha this went away because it is now mentinoed in deployment/env-vars.md?

Reply:
I guess so, yes 😄 I moved things around so it made sense when reading the docs.